app/styles/onboarding-card.module.css

```diff
@@ -1,19 +1,95 @@
 .onboarding-card{
-    min-height: 60vh;
+    min-height: 55vh;
     min-width: 40vw;
     display: flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    border: 2px solid var(--color-lightgrey);
-    background-color: var(--color-cultured);
-    border-radius: 10px;
-    margin-top: 80px;
+    background-color: #FFF;
+    border-radius: 24px;
+    position: relative;
+    padding: 0px 16px;
+    box-shadow: 0px 4px 10px -2px rgba(16, 24, 40, 0.06), 0px 6px 15px -2px rgba(16, 24, 40, 0.10);
 }
-@media screen and (max-width: 625px) {
+
+.signup-details__rds-logo{
+    display: block;
+    position: absolute;
+    transform: translateX(-50%);
+    left: 50%;
+    top: -10%;
+}
+
+.logos-container{
+    padding: 25px;
+    display: flex;
+    flex-direction: row;
+    position: absolute;
+    transform: translateX(-50%);
+    left: 50%;
+    top: 3%;
+}
```

Comment:
NIT - add line spacing here

Reply:
@vinit717 will work on this step-0 card in future.